src/foam/demos/examples/FBE.js

```diff
@@ -169,8 +169,8 @@ foam.CLASS({
           var self = this;
           this.dom.removeAllChildren();
           var scope = {
-            el: function() {
-              return self.dom.el();
+            el: async function() {
```

Comment:
I think these two changes have no affect. It will still return a promise.

Reply:
Not used anyway, so I think we can just remove.

Reply:
Removed it.